man/systemd.exec.xml

```diff
@@ -656,6 +656,32 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         to 0022.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>CoredumpFilter=</varname></term>
+
+        <listitem><para>Controls which types of memory mappings will be saved if the process dumps core
+        (using the <filename>/proc/<replaceable>pid</replaceable>/coredump_filter</filename> file). Takes a
+        whitespace-separated combination of mapping type names or numbers (with the default base 16). Mapping
```

Comment:
code suggests it's to base 10 (which we normally don't mention)

Reply:
Hmm, the code uses atoux64 (in the latest iteration, and atoux16 in the first one). And there's a test case in test-coredump-util. So unless I missed something, it should be using base 16.

It is supposed to use base 16 because that's what the kernel uses in `coredump_filter`.

Reply:
the 16 or 64 refers to the bit width of the parsed value, not the base used in the notation.

safe_ato*() all parse decimal by default (but understand 0x prefixes in the string to switch to hex parsing)

Reply:
Not all. safe_atoux16() and the newly added safe_atoux64 use base 16 if not specified.

Reply:
oh, i am an idiot, didn't see the "x" instead of the "u", ignore me